ts/features/fci/screens/valid/FciDataSharingScreen.tsx

```diff
@@ -0,0 +1,182 @@
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { constNull } from "fp-ts/lib/function";
+import { List } from "native-base";
+import * as React from "react";
+import { SafeAreaView, StyleSheet, View, Text } from "react-native";
+import { IOColors } from "../../../../components/core/variables/IOColors";
+import BaseScreenComponent from "../../../../components/screens/BaseScreenComponent";
+import ListItemComponent from "../../../../components/screens/ListItemComponent";
+import ScreenContent from "../../../../components/screens/ScreenContent";
+import FooterWithButtons from "../../../../components/ui/FooterWithButtons";
+import IconFont from "../../../../components/ui/IconFont";
+import I18n from "../../../../i18n";
+import { useIOSelector } from "../../../../store/hooks";
+import {
+  profileFiscalCodeSelector,
+  profileNameSelector,
+  profileSelector
+} from "../../../../store/reducers/profile";
+import customVariables from "../../../../theme/variables";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+import { capitalize } from "../../../../utils/strings";
+import {
+  cancelButtonProps,
+  confirmButtonProps
+} from "../../../bonus/bonusVacanze/components/buttons/ButtonConfigurations";
+import { useFciAbortSignatureFlow } from "../../hooks/useFciAbortSignatureFlow";
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1
+  },
+  topSubtitle: {
+    fontSize: customVariables.fontSizeBase,
+    fontFamily: customVariables.fontFamily,
+    lineHeight: customVariables.lineHeightBase,
+    color: customVariables.textColor
+  },
+  topScreenContainer: {
+    flex: 1,
+    justifyContent: "flex-end"
+  },
+  padded: {
+    paddingLeft: customVariables.contentPadding,
+    paddingRight: customVariables.contentPadding,
+    paddingBottom: customVariables.contentPadding
+  },
+  verticalPadding: {
+    paddingTop: customVariables.spacerHeight,
+    paddingBottom: customVariables.spacerHeight
+  },
+  linkDecoration: {
+    textDecorationLine: "underline",
+    color: IOColors.blue
+  },
+  paddingTextLarge: {
+    paddingLeft: 14
+  },
+  paddingText: {
+    paddingLeft: 4
+  },
+  alertTextContainer: {
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center"
+  },
+  containerTitle: {
+    paddingTop: customVariables.contentPadding,
+    paddingBottom: customVariables.contentPadding
+  },
+  bottomPadding: { paddingBottom: 20 }
+});
+
+const iconSize = 24;
+
+const FciDataSharingScreen = (): React.ReactElement => {
```

Comment:
One general question on the Screen component:

Why using bare Text component instead of Typographical defined componets?